package/src/skia/types/Canvas.ts

```diff
@@ -492,4 +493,23 @@ export interface SkCanvas {
    * @param skp
    */
   drawPicture(skp: SkPicture): void;
+
+  /** Read Image pixels
+   *
+   * @param srcX - x-axis upper left corner of the rectangle to read from
+   * @param srcY - y-axis upper left corner of the rectangle to read from
+   * @param imageInfo - describes the pixel format and dimensions of the data to read into
+   * @param dest - optional array to read pixels into. Must be large enough to hold all pixels in imageInfo.
+   * @param bytesPerRow - optional number of bytes per row to read.
+   *                      If not provided, imageInfo.width * imageInfo.bytesPerPixel will be used.
+   *
+   * @return Float32Array or Uint8Array with data
+   */
+  readPixels(
+    srcX: number,
+    srcY: number,
+    imageInfo: ImageInfo,
+    dest?: Float32Array | Uint8Array | MallocObj,
```

Comment:
MallocObj can be removed here

Reply:
The `MallocObj` is for web.

https://github.com/google/skia/blob/d756a2f5665dc36024488838ecd98c0f04073c07/modules/canvaskit/npm_build/types/index.d.ts#L2044
Their implement is use WASM memory space and copy(convert TypedArray), instead of write data into TypedArray directly (my JSI implement).